examples/product/config/sw-config.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
       handler: "fastest",
       urlPattern: "^https:\/\/fonts\.gstatic\.com"
     }],
-    staticFileGlobs: ['dist/**/*']
+    staticFileGlobs: ['dist/js/*.{js,css,png}']
```

Comment:
I updated this glob so it wouldn't match the icons folder. Since there's like a dozen or more icons there, and no browser is every going to request *all* of them, pre-caching them just means that we're requesting a lot of icons that wont even be used.